### PR TITLE
Change error message and success flash content

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2815

## Screenshots of UI changes:

Error message:

Screenshot 2021-07-21 at 16 51 56

Success notification thing:

Screenshot 2021-07-21 at 17 04 45

### CODE REVIEW

File: config/locales/jobseekers.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ en:
         success: Thank you for your feedback on your job alert
       update:
         success: Your further feedback on job alerts has been sent to the team
+    job_aler_further_feedbacks:
```
Comment (alexbowen) [inline config/locales/jobseekers.yml:43]: typo aler

Comment (josephhull676) [reply config/locales/jobseekers.yml:43]: Ooo

Comment (josephhull676) [reply config/locales/jobseekers.yml:43]: It's also just straight-up wrong lol

Comment (alexbowen) [reply config/locales/jobseekers.yml]: yeah, have you checked something isnt amis where this translation is used as it was straight up wrong??

Comment (josephhull676) [reply config/locales/jobseekers.yml]: Fixed now!

**Review by alexbowen — APPROVED** [review]
just left comment...

**Pull request merged**